vulnerabilities/views.py

```diff
@@ -96,17 +104,34 @@ class VulnerabilitySearch(ListView):
     ordering = ["vulnerability_id"]
     paginate_by = PAGE_SIZE
 
+    def get_paginate_by(self, queryset):
+        page_size = self.request.GET.get("page_size", "")
```

Comment:
The user provide value for the page_size should be validated to prevent a user from provide a too large number.